damus/Views/SideMenuView.swift

```diff
@@ -158,6 +158,9 @@ struct SideMenuView: View {
                 isSidebarVisible.toggle()
             }
             .alert("Logout", isPresented: $confirm_logout) {
```

Comment:
It looks like it's recommended to use an action sheet in this case:

https://developer.apple.com/design/human-interface-guidelines/components/presentation/action-sheets

Reply:
I think it might be this?
https://developer.apple.com/documentation/swiftui/view/confirmationdialog(_:ispresented:titlevisibility:actions:message:)-2s7pz